frontend/src/scenes/paths/pathsLogic.ts

```diff
@@ -26,14 +26,16 @@ function cleanPathParams(filters: Partial<FilterType>): Partial<FilterType> {
         end_point: filters.end_point,
         // TODO: use FF for path_type undefined
         path_type: filters.path_type ? filters.path_type || PathType.PageView : undefined,
-        include_event_types: filters.include_event_types || [PathType.PageView],
+        include_event_types: filters.include_event_types || [...(filters.funnel_filter ? [] : [PathType.PageView])],
```

Comment:
Unless I'm misunderstanding this:

```suggestion
        include_event_types: filters.include_event_types || (filters.funnel_filter ? [] : [PathType.PageView]),
```